beast/observationmodel/ast/make_ast_xy_list.py

```diff
@@ -120,61 +121,75 @@ def pick_positions_from_map(
     # if appropriate information is given, extract the x/y positions so that
     # there are no ASTs generated outside of the catalog footprint
     colnames = catalog.data.columns
+    xy_pos = False
+    radec_pos = False
 
+    # if x/y in catalog, save them
     if "X" or "x" in colnames:
```

Comment:
This probably should be: if ("X" in colnames) or ("x" in colnames).  I think the current version evaluates if ("X") or ("x" in colnames).  So should always be true as "X" is always true.  But maybe I don't understand how the *in* operator works.

Reply:
I changed this!